### PR TITLE
[Feature-#5273][server-master] Task node of SWITCH 

### PR DESCRIPTION
**Pull request by wangxj3**
[Feature-#5273][server-master] Task node of SWITCH 
Determine the nodes that need to be run based on the value of the parameter.

### CODE REVIEW

**Comment by CalvinKirs** [comment]
sonar check fail,please reslove.

File: dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/TaskParametersUtils.java [hunk 1]
```diff
@@ -88,6 +89,8 @@ public static AbstractParameters getParameters(String taskType, String parameter
                 return JSONUtils.parseObject(parameter, ConditionsParameters.class);
             case SQOOP:
                 return JSONUtils.parseObject(parameter, SqoopParameters.class);
+            case SWITCH:
+                return JSONUtils.parseObject(parameter, SwitchParameters.class);
```
Comment (brave-lee) [inline dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/TaskParametersUtils.java:93]: Please use new code of dev

Comment (wangxj3) [reply dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/TaskParametersUtils.java:93]: I fix the globalParam , then I will fix this .

**Comment by CalvinKirs** [comment]
Sorry, a review is so late. can you resolve the conflicts file?

**Comment by wangxj3** [comment]
> 
> 
> Sorry, a review is so late. can you resolve the conflicts file?

This function relies on global params, so I will change it after the global params is successfully committed

**Pull request closed without merging**